sky/provision/kubernetes/utils.py

```diff
@@ -131,6 +134,11 @@ def wrapper(*args, **kwargs):
             backoff = common_utils.Backoff(initial_backoff=retry_interval,
                                            max_backoff_factor=3)
 
+            # try to capture the kubernetes context from parameters
+            if context_param:
+                context = kwargs.get(context_param)
+                assert context is not None, f'{context_param} is required'
```

Comment:
How about we remove the `context_param` as well, and just assert that `assert 'context' in kwargs`?